src/utils/transaction.js

```diff
@@ -199,22 +182,37 @@ export default {
     prepareIssueAndBurn: function(amountData) {
         var tokenIdx = transferInt()
         var amountArr = transferAmount(amountData)
-        var encodeArr = amountArr.concat(tokenIdx)
-        return base58_1.default.encode(encodeArr);
+
+        var typeArr = new Array(2)
+        typeArr[0] = 0 & (255)
+        typeArr[1] = 2 & (255)
+
+        var encodeArr = typeArr.concat(amountArr.concat(tokenIdx))
+        return base58_1.default.encode(Uint8Array.from(encodeArr));
     },
     prepareCreate: function(max, unity, tokenDescription) {
         var maxArr = transferAmount(max)
         var unityArr = transferAmount(unity)
         var desArr = transferShortTxt(tokenDescription)
-        var encodeArr = maxArr.concat(unityArr.concat(desArr))
-        return base58_1.default.encode(encodeArr);
+
+        var typeArr = new Array(2)
+        typeArr[0] = 0 & (255)
+        typeArr[1] = 3 & (255)
+
+        var encodeArr = typeArr.concat(maxArr.concat(unityArr.concat(desArr)))
+        return base58_1.default.encode(Uint8Array.from(encodeArr));
     },
     prepareSend: function(recipient, amount) {
         var accountArr = transferAccount(recipient)
         var tokenIdx = transferInt()
```

Comment:
Though `tokenIdx`  is fixed value in this version, it'd better change to 
```
var tokenIdx = 0
var tokenIdxArr = transferInt(tokenIdx)
```
The function `transferInt` should accept an integer and return 4 bytes.